Gruntfile.js

```diff
@@ -56,8 +56,8 @@ module.exports = function(grunt) {
           vendorRoot + "moment/min/moment.min.js",
           vendorRoot + "moment-timezone/builds/moment-timezone-with-data.min.js",
           vendorRoot + "adminlte/dist/js/app.min.js",
-          "src/static/extra/ui-bootstrap-custom-0.13.0.min.js",
-          "src/static/extra/ui-bootstrap-custom-tpls-0.13.0.min.js"
```

Comment:
can you revert this and pull in the latest changes?

The reason I have used a custom build of the ui-bootstrap is because the whole ui-bootstrap is very heavy and I'm currently using only pagination. There was a bug where these files weren't being loaded in the dev builds I've fixed this as well.